lists/ke.csv

```diff
@@ -123,7 +123,7 @@ https://www.the-star.co.ke,NEWS,News Media,2016-06-13,OONI,
 http://www.the-star.co.ke/news/2015/12/11/ti-bribes-report-shows-kenya-police-judiciary-among-most-corrupt_c1259309,POLR,Political Criticism,2016-06-13,OONI,
 http://www.theeastafrican.co.ke,NEWS,News Media,2014-04-15,citizenlab,
 http://www.tikenya.org,POLR,Political Criticism,2014-04-15,citizenlab,
-"http://www.time.com/time/world/article/0,8599,2057966,00.html",HUMR,Human Rights Issues,2014-04-15,citizenlab,

```

Comment:
Since there are commas in the URL, it must be in quotation marks to avoid breaking the CSV